### PR TITLE
Changes on issue comment alert title and actions

### LINKED ISSUE #826: Add “user’s comment” to alert title and simplify actions
GitHawk Upload by rnystrom

<details>
<summary>Bug Report Dump (Auto-generated)</summary>
<pre>
Version 1.12.0 (1137)
Device: iPhone 7 Plus (iOS 11.0.3)
TestFlight: false
</pre>
</details>

### PR DESCRIPTION
**Pull request by rizwankce**
Closes #826 #801

### CODE REVIEW

File: Classes/Issues/Comments/IssueCommentSectionController.swift [hunk 1]
```diff
@@ -297,7 +297,9 @@ EditCommentViewControllerDelegate {
     // MARK: IssueCommentDetailCellDelegate
 
     func didTapMore(cell: IssueCommentDetailCell, sender: UIView) {
-        let alert = UIAlertController.configured(preferredStyle: .actionSheet)
+        guard let login = object?.details.login else { return }
+
+        let alert = UIAlertController.configured(title: "\(login)'s comment", preferredStyle: .actionSheet)
```
Comment (BasThomas) [inline Classes/Issues/Comments/IssueCommentSectionController.swift]: `NSLocalizedString`?

File: Classes/Issues/Comments/IssueCommentSectionController.swift [hunk 2]
```diff
@@ -297,7 +297,9 @@ EditCommentViewControllerDelegate {
     // MARK: IssueCommentDetailCellDelegate
 
     func didTapMore(cell: IssueCommentDetailCell, sender: UIView) {
-        let alert = UIAlertController.configured(preferredStyle: .actionSheet)
+        guard let login = object?.details.login else { return }
```
Comment (BasThomas) [inline Classes/Issues/Comments/IssueCommentSectionController.swift]: Do we want to fail completely (and silently) if we don't have a `login`?

Comment (rizwankce) [reply Classes/Issues/Comments/IssueCommentSectionController.swift]: `didTapProfile` also has the same bahaviour!

Comment (Sherlouk) [reply Classes/Issues/Comments/IssueCommentSectionController.swift]: I'd still say let's just a quick toast error in before the return! 😅 I'd rather have _some_ indication of failure

Comment (rizwankce) [reply Classes/Issues/Comments/IssueCommentSectionController.swift]: Added Generic error toast

Comment (rnystrom) [reply Classes/Issues/Comments/IssueCommentSectionController.swift]: I wonder if we could make a `guardElseToast` function 😝

Comment (BasThomas) [reply Classes/Issues/Comments/IssueCommentSectionController.swift]: We can at least just `return ToastManager.showGenericError()` inside the else?

**Pull request merged**